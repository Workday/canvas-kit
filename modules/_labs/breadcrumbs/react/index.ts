import Breadcrumbs from './lib/Breadcrumbs';

export default Breadcrumbs;
export {Breadcrumbs};
export * from './lib/Breadcrumbs';
