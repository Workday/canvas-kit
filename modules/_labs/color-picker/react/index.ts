export {default, default as ColorPicker, ColorPickerProps} from './lib/ColorPicker';
export {Swatch} from './lib/parts/Swatch';
