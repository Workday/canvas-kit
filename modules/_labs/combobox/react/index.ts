import Combobox from './lib/Combobox';

export default Combobox;
export {Combobox};
export * from './lib/Combobox';
