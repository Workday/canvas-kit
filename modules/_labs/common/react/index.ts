export * from './lib/Flex';
export * from './lib/Box';
