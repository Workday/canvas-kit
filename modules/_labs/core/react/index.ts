import type from './lib/type';
import space from './lib/space';
export default type;
export {type, space};
export * from './lib/type';
export * from './lib/StaticStates';
