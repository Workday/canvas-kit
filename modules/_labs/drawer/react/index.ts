import Drawer from './lib/Drawer';
import DrawerHeader from './lib/DrawerHeader';
export {Drawer, DrawerHeader};
export default Drawer;
export * from './lib/Drawer';
