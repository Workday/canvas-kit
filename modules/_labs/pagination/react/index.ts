import Pagination from './lib/Pagination';

export default Pagination;
export {Pagination};
export * from './lib/Pagination';
