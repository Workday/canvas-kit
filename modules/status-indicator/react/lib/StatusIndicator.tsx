import * as React from 'react';
import {SystemIcon} from '@workday/canvas-kit-react-icon';
import {CanvasSystemIcon} from '@workday/design-assets-types';
import {GenericStyle} from '@workday/canvas-kit-react-common';
import {borderRadius, colors, type, spacing} from '@workday/canvas-kit-react-core';
import {CSSObject} from '@emotion/core';
import styled from '@emotion/styled';

export enum StatusIndicatorType {
  Gray = 'gray',
  Orange = 'orange',
  Blue = 'blue',
  Green = 'green',
  Red = 'red',
  Transparent = 'transparent',
}

export enum StatusIndicatorEmphasis {
  High = 'high',
  Low = 'low',
}

export interface StatusIndicatorGenericStyle extends GenericStyle {
  styles: CSSObject & React.CSSProperties;
  variants: {
    [statusType in StatusIndicatorType]: {
      [statusEmphasis in StatusIndicatorEmphasis]?: CSSObject & React.CSSProperties;
    };
  };
}

export const statusIndicatorStyles: StatusIndicatorGenericStyle = {
  classname: 'status-indicator',
  styles: {
    ...type.small,
    display: 'inline-flex',
    alignItems: 'center',
    verticalAlign: 'middle',
    maxWidth: 150,
    height: spacing.s,
    padding: `1px ${spacing.xxxs}`,
    borderRadius: borderRadius.s,
    boxSizing: 'border-box',
  },
  variants: {
    gray: {
      high: {
        color: colors.frenchVanilla100,
        background: colors.licorice300,
      },
      low: {
        color: colors.licorice400,
        background: colors.soap200,
      },
    },
    orange: {
      high: {
        color: colors.licorice500,
        background: colors.cantaloupe400,
      },
      low: {
        color: colors.toastedMarshmallow600,
        background: colors.cantaloupe100,
      },
    },
    blue: {
      high: {
        color: colors.frenchVanilla100,
        background: colors.blueberry400,
      },
      low: {
        color: colors.blueberry500,
        background: colors.blueberry100,
      },
    },
    green: {
      high: {
        color: colors.frenchVanilla100,
        background: colors.greenApple600,
      },
      low: {
        color: colors.greenApple600,
        background: colors.greenApple100,
      },
    },
    red: {
      high: {
        color: colors.frenchVanilla100,
        background: colors.cinnamon500,
      },
      low: {
        color: colors.cinnamon600,
        background: colors.cinnamon100,
      },
    },
    transparent: {
      high: {
        color: colors.frenchVanilla100,
        background: colors.blackPepper600,
      },
      // Low & High emphasis are identical for transparent status indicators
      low: {
        color: colors.frenchVanilla100,
        background: colors.blackPepper600,
      },
    },
  },
};

export interface StatusIndicatorProps extends React.HTMLAttributes<HTMLSpanElement> {
  /**
   * Type of status indicator (e.g. Gray, Orange etc.).
   */
  type: StatusIndicatorType;
  /**
   * Emphasis of status indicator (e.g. High v Low).
   */
  emphasis: StatusIndicatorEmphasis;
  /**
   * Label of status indicator.
   */
  label: string;
  /**
   * Icon to be used along with the status label.
   */
  icon?: CanvasSystemIcon;
}

const Container = styled('span')<Pick<StatusIndicatorProps, 'type' | 'emphasis'>>(
  statusIndicatorStyles.styles,
  ({type, emphasis}) => ({
    ...statusIndicatorStyles.variants[type][emphasis],
  })
);

const StatusLabel = styled('span')({
  fontWeight: 'bold',
  '-webkit-font-smoothing': 'antialiased',
  textTransform: 'uppercase',
  overflow: 'hidden',
  whiteSpace: 'nowrap',
  textOverflow: 'ellipsis',
});

export default class StatusIndicator extends React.Component<StatusIndicatorProps> {
  public static Type = StatusIndicatorType;
  public static Emphasis = StatusIndicatorEmphasis;
  static defaultProps = {
    emphasis: StatusIndicatorEmphasis.High,
  };

  public render() {
    const {type, emphasis, icon, label, ...elemProps} = this.props;
    const variant = statusIndicatorStyles.variants[type][emphasis]!;

    return (
      <Container type={type} emphasis={emphasis} {...elemProps}>
        {icon && (
          <SystemIcon
            colorHover={variant.color}
            color={variant.color}
            icon={icon}
            size={14}
            style={{paddingRight: spacing.xxxs}}
          />
        )}
        <StatusLabel>{label}</StatusLabel>
      </Container>
    );
  }
}
