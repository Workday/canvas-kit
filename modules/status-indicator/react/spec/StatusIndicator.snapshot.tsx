import * as React from 'react';
import * as renderer from 'react-test-renderer';
import StatusIndicator from '../lib/StatusIndicator';
import {uploadCloudIcon} from '@workday/canvas-system-icons-web';

describe('StatusIndicator Snapshots', () => {
  test('renders a StatusIndicator with high emphasis', () => {
    const component = renderer.create(
      <StatusIndicator type={StatusIndicator.Type.Green} label={'Test Label'} />
    );
    expect(component).toMatchSnapshot();
  });
  test('renders a StatusIndicator with low emphasis', () => {
    const component = renderer.create(
      <StatusIndicator
        type={StatusIndicator.Type.Blue}
        emphasis={StatusIndicator.Emphasis.Low}
        label={'Test Label'}
      />
    );
    expect(component).toMatchSnapshot();
  });
  test('renders a StatusIndicator with no low emphasis config, falls back to default', () => {
    const component = renderer.create(
      <StatusIndicator
        type={StatusIndicator.Type.Transparent}
        emphasis={StatusIndicator.Emphasis.Low}
        label={'Test Label'}
      />
    );
    expect(component).toMatchSnapshot();
  });
  test('renders a StatusIndicator with high emphasis and icon', () => {
    const component = renderer.create(
      <StatusIndicator
        type={StatusIndicator.Type.Transparent}
        label={'Test Label'}
        icon={uploadCloudIcon}
      />
    );
    expect(component).toMatchSnapshot();
  });
  test('renders a StatusIndicator with low emphasis and icon', () => {
    const component = renderer.create(
      <StatusIndicator
        type={StatusIndicator.Type.Red}
        label={'Test Label'}
        icon={uploadCloudIcon}
      />
    );
    expect(component).toMatchSnapshot();
  });
  test('renders a StatusIndicator with no low emphasis config and icon, falls back to default', () => {
    const component = renderer.create(
      <StatusIndicator
        type={StatusIndicator.Type.Transparent}
        emphasis={StatusIndicator.Emphasis.Low}
        label={'Test Label'}
        icon={uploadCloudIcon}
      />
    );
    expect(component).toMatchSnapshot();
  });
});
