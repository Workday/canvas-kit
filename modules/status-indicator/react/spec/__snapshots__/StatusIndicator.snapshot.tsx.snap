// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StatusIndicator Snapshots renders a StatusIndicator with high emphasis 1`] = `
.emotion-2 {
  font-size: 12px;
  line-height: 16px;
  font-weight: 400;
  color: #494949;
  font-family: "Roboto","Helvetica Neue","Helvetica",Arial,sans-serif;
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  vertical-align: middle;
  max-width: 150px;
  height: 16px;
  padding: 1px 4px;
  border-radius: 2px;
  box-sizing: border-box;
  color: #ffffff;
  background: #217a37;
}

.emotion-0 {
  font-weight: bold;
  -webkit-font-smoothing: antialiased;
  text-transform: uppercase;
  overflow: hidden;
  white-space: nowrap;
  text-overflow: ellipsis;
}

<span
  className="emotion-2 emotion-3"
  type="green"
>
  <span
    className="emotion-0 emotion-1"
  >
    Test Label
  </span>
</span>
`;

exports[`StatusIndicator Snapshots renders a StatusIndicator with high emphasis and icon 1`] = `
.emotion-1 {
  font-weight: bold;
  -webkit-font-smoothing: antialiased;
  text-transform: uppercase;
  overflow: hidden;
  white-space: nowrap;
  text-overflow: ellipsis;
}

.emotion-3 {
  font-size: 12px;
  line-height: 16px;
  font-weight: 400;
  color: #494949;
  font-family: "Roboto","Helvetica Neue","Helvetica",Arial,sans-serif;
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  vertical-align: middle;
  max-width: 150px;
  height: 16px;
  padding: 1px 4px;
  border-radius: 2px;
  box-sizing: border-box;
  color: #ffffff;
  background: #000000;
}

.emotion-0 {
  display: inline-block;
}

.emotion-0 svg {
  display: block;
}

.emotion-0 .wd-icon-fill {
  fill: #ffffff;
}

.emotion-0:hover .wd-icon-fill {
  fill: #ffffff;
}

.emotion-0 .wd-icon-accent {
  fill: #ffffff;
}

.emotion-0:hover .wd-icon-accent {
  fill: #ffffff;
}

.emotion-0 .wd-icon-background {
  fill: transparent;
}

.emotion-0:hover .wd-icon-background {
  fill: transparent;
}

.emotion-0 svg {
  width: 14px;
  height: 14px;
}

<span
  className="emotion-3 emotion-4"
  type="transparent"
>
  <span
    className="emotion-0"
    dangerouslySetInnerHTML={
      Object {
        "__html": "<svg xmlns=\\"http://www.w3.org/2000/svg\\" width=\\"24\\" height=\\"24\\" class=\\"wd-icon-upload-cloud wd-icon\\" focusable=\\"false\\" role=\\"presentation\\" viewBox=\\"0 0 24 24\\"><g fill-rule=\\"evenodd\\" class=\\"wd-icon-container\\"><path fill-rule=\\"nonzero\\" d=\\"M2 13.864c0-1.565.912-2.944 2.257-3.545C4.381 7.367 6.737 5 9.642 5c1.56 0 3.002.69 4.007 1.842a3.608 3.608 0 0 1 1.753-.454c1.862 0 3.392 1.41 3.659 3.238C20.766 10.11 22 11.718 22 13.603c0 2.395-1.827 5.033-4.005 5.033H5.754C3.705 18.636 2 16.113 2 13.864zm4.084-2.97l.067.797-.783.167c-.89.19-1.55 1.022-1.55 2.006 0 1.36 1.077 2.954 1.936 2.954h12.241c1.002 0 2.187-1.712 2.187-3.215 0-1.228-.91-2.23-2.053-2.301l-.909-.058.06-.909c.002-.046.004-.093.004-.14 0-1.106-.85-1.989-1.882-1.989-.49 0-.95.199-1.3.552l-.827.836-.601-1.01c-.65-1.093-1.788-1.766-3.032-1.766-1.965 0-3.571 1.669-3.571 3.744 0 .112.004.222.013.332zM13 13.814v3.006h-2v-2.992L9.77 15.06a.497.497 0 0 1-.713-.002l-.7-.7a.496.496 0 0 1-.002-.712L12 10l3.66 3.645a.497.497 0 0 1-.003.712l-.7.7a.496.496 0 0 1-.712.002L13 13.814z\\" class=\\"wd-icon-fill\\"/></g></svg>",
      }
    }
    style={
      Object {
        "paddingRight": "4px",
      }
    }
  />
  <span
    className="emotion-1 emotion-2"
  >
    Test Label
  </span>
</span>
`;

exports[`StatusIndicator Snapshots renders a StatusIndicator with low emphasis 1`] = `
.emotion-0 {
  font-weight: bold;
  -webkit-font-smoothing: antialiased;
  text-transform: uppercase;
  overflow: hidden;
  white-space: nowrap;
  text-overflow: ellipsis;
}

.emotion-2 {
  font-size: 12px;
  line-height: 16px;
  font-weight: 400;
  color: #494949;
  font-family: "Roboto","Helvetica Neue","Helvetica",Arial,sans-serif;
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  vertical-align: middle;
  max-width: 150px;
  height: 16px;
  padding: 1px 4px;
  border-radius: 2px;
  box-sizing: border-box;
  color: #005cb9;
  background: #d7eafc;
}

<span
  className="emotion-2 emotion-3"
  type="blue"
>
  <span
    className="emotion-0 emotion-1"
  >
    Test Label
  </span>
</span>
`;

exports[`StatusIndicator Snapshots renders a StatusIndicator with low emphasis and icon 1`] = `
.emotion-1 {
  font-weight: bold;
  -webkit-font-smoothing: antialiased;
  text-transform: uppercase;
  overflow: hidden;
  white-space: nowrap;
  text-overflow: ellipsis;
}

.emotion-0 {
  display: inline-block;
}

.emotion-0 svg {
  display: block;
}

.emotion-0 .wd-icon-fill {
  fill: #ffffff;
}

.emotion-0:hover .wd-icon-fill {
  fill: #ffffff;
}

.emotion-0 .wd-icon-accent {
  fill: #ffffff;
}

.emotion-0:hover .wd-icon-accent {
  fill: #ffffff;
}

.emotion-0 .wd-icon-background {
  fill: transparent;
}

.emotion-0:hover .wd-icon-background {
  fill: transparent;
}

.emotion-0 svg {
  width: 14px;
  height: 14px;
}

.emotion-3 {
  font-size: 12px;
  line-height: 16px;
  font-weight: 400;
  color: #494949;
  font-family: "Roboto","Helvetica Neue","Helvetica",Arial,sans-serif;
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  vertical-align: middle;
  max-width: 150px;
  height: 16px;
  padding: 1px 4px;
  border-radius: 2px;
  box-sizing: border-box;
  color: #ffffff;
  background: #de2e21;
}

<span
  className="emotion-3 emotion-4"
  type="red"
>
  <span
    className="emotion-0"
    dangerouslySetInnerHTML={
      Object {
        "__html": "<svg xmlns=\\"http://www.w3.org/2000/svg\\" width=\\"24\\" height=\\"24\\" class=\\"wd-icon-upload-cloud wd-icon\\" focusable=\\"false\\" role=\\"presentation\\" viewBox=\\"0 0 24 24\\"><g fill-rule=\\"evenodd\\" class=\\"wd-icon-container\\"><path fill-rule=\\"nonzero\\" d=\\"M2 13.864c0-1.565.912-2.944 2.257-3.545C4.381 7.367 6.737 5 9.642 5c1.56 0 3.002.69 4.007 1.842a3.608 3.608 0 0 1 1.753-.454c1.862 0 3.392 1.41 3.659 3.238C20.766 10.11 22 11.718 22 13.603c0 2.395-1.827 5.033-4.005 5.033H5.754C3.705 18.636 2 16.113 2 13.864zm4.084-2.97l.067.797-.783.167c-.89.19-1.55 1.022-1.55 2.006 0 1.36 1.077 2.954 1.936 2.954h12.241c1.002 0 2.187-1.712 2.187-3.215 0-1.228-.91-2.23-2.053-2.301l-.909-.058.06-.909c.002-.046.004-.093.004-.14 0-1.106-.85-1.989-1.882-1.989-.49 0-.95.199-1.3.552l-.827.836-.601-1.01c-.65-1.093-1.788-1.766-3.032-1.766-1.965 0-3.571 1.669-3.571 3.744 0 .112.004.222.013.332zM13 13.814v3.006h-2v-2.992L9.77 15.06a.497.497 0 0 1-.713-.002l-.7-.7a.496.496 0 0 1-.002-.712L12 10l3.66 3.645a.497.497 0 0 1-.003.712l-.7.7a.496.496 0 0 1-.712.002L13 13.814z\\" class=\\"wd-icon-fill\\"/></g></svg>",
      }
    }
    style={
      Object {
        "paddingRight": "4px",
      }
    }
  />
  <span
    className="emotion-1 emotion-2"
  >
    Test Label
  </span>
</span>
`;

exports[`StatusIndicator Snapshots renders a StatusIndicator with no low emphasis config and icon, falls back to default 1`] = `
.emotion-1 {
  font-weight: bold;
  -webkit-font-smoothing: antialiased;
  text-transform: uppercase;
  overflow: hidden;
  white-space: nowrap;
  text-overflow: ellipsis;
}

.emotion-3 {
  font-size: 12px;
  line-height: 16px;
  font-weight: 400;
  color: #494949;
  font-family: "Roboto","Helvetica Neue","Helvetica",Arial,sans-serif;
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  vertical-align: middle;
  max-width: 150px;
  height: 16px;
  padding: 1px 4px;
  border-radius: 2px;
  box-sizing: border-box;
  color: #ffffff;
  background: #000000;
}

.emotion-0 {
  display: inline-block;
}

.emotion-0 svg {
  display: block;
}

.emotion-0 .wd-icon-fill {
  fill: #ffffff;
}

.emotion-0:hover .wd-icon-fill {
  fill: #ffffff;
}

.emotion-0 .wd-icon-accent {
  fill: #ffffff;
}

.emotion-0:hover .wd-icon-accent {
  fill: #ffffff;
}

.emotion-0 .wd-icon-background {
  fill: transparent;
}

.emotion-0:hover .wd-icon-background {
  fill: transparent;
}

.emotion-0 svg {
  width: 14px;
  height: 14px;
}

<span
  className="emotion-3 emotion-4"
  type="transparent"
>
  <span
    className="emotion-0"
    dangerouslySetInnerHTML={
      Object {
        "__html": "<svg xmlns=\\"http://www.w3.org/2000/svg\\" width=\\"24\\" height=\\"24\\" class=\\"wd-icon-upload-cloud wd-icon\\" focusable=\\"false\\" role=\\"presentation\\" viewBox=\\"0 0 24 24\\"><g fill-rule=\\"evenodd\\" class=\\"wd-icon-container\\"><path fill-rule=\\"nonzero\\" d=\\"M2 13.864c0-1.565.912-2.944 2.257-3.545C4.381 7.367 6.737 5 9.642 5c1.56 0 3.002.69 4.007 1.842a3.608 3.608 0 0 1 1.753-.454c1.862 0 3.392 1.41 3.659 3.238C20.766 10.11 22 11.718 22 13.603c0 2.395-1.827 5.033-4.005 5.033H5.754C3.705 18.636 2 16.113 2 13.864zm4.084-2.97l.067.797-.783.167c-.89.19-1.55 1.022-1.55 2.006 0 1.36 1.077 2.954 1.936 2.954h12.241c1.002 0 2.187-1.712 2.187-3.215 0-1.228-.91-2.23-2.053-2.301l-.909-.058.06-.909c.002-.046.004-.093.004-.14 0-1.106-.85-1.989-1.882-1.989-.49 0-.95.199-1.3.552l-.827.836-.601-1.01c-.65-1.093-1.788-1.766-3.032-1.766-1.965 0-3.571 1.669-3.571 3.744 0 .112.004.222.013.332zM13 13.814v3.006h-2v-2.992L9.77 15.06a.497.497 0 0 1-.713-.002l-.7-.7a.496.496 0 0 1-.002-.712L12 10l3.66 3.645a.497.497 0 0 1-.003.712l-.7.7a.496.496 0 0 1-.712.002L13 13.814z\\" class=\\"wd-icon-fill\\"/></g></svg>",
      }
    }
    style={
      Object {
        "paddingRight": "4px",
      }
    }
  />
  <span
    className="emotion-1 emotion-2"
  >
    Test Label
  </span>
</span>
`;

exports[`StatusIndicator Snapshots renders a StatusIndicator with no low emphasis config, falls back to default 1`] = `
.emotion-0 {
  font-weight: bold;
  -webkit-font-smoothing: antialiased;
  text-transform: uppercase;
  overflow: hidden;
  white-space: nowrap;
  text-overflow: ellipsis;
}

.emotion-2 {
  font-size: 12px;
  line-height: 16px;
  font-weight: 400;
  color: #494949;
  font-family: "Roboto","Helvetica Neue","Helvetica",Arial,sans-serif;
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  vertical-align: middle;
  max-width: 150px;
  height: 16px;
  padding: 1px 4px;
  border-radius: 2px;
  box-sizing: border-box;
  color: #ffffff;
  background: #000000;
}

<span
  className="emotion-2 emotion-3"
  type="transparent"
>
  <span
    className="emotion-0 emotion-1"
  >
    Test Label
  </span>
</span>
`;
